grant/grant_test.go

```diff
@@ -137,5 +142,19 @@ func TestVerifyAndConsume(t *testing.T) {
 		t.Error("expected re-redeem with the same wallet and promotion to fail")
 	}
 
+	decodedGrants, err := DecodeGrants(redeemedGrant)
+	if err != nil {
+		t.Error("Unable to decode grants")
+	}
+	redeemedIDs, err := GetRedeemedIDs(ctx, decodedGrants)
+	if err != nil {
+		t.Error("Unable to check RedeemedIDs")
+	}
+	expectedRedeemedIDs := []string{grantID}
+	fmt.Println(redeemedIDs)
```

Comment:
can you remove this, I think debugging need is covered by error case you have below